Documentation/data-collection.md

```diff
@@ -41,32 +41,32 @@ data:
     # available. This is used to ensure that updates are being properly
     # served to clusters.
     - '{__name__="cluster_version_available_updates"}'
-    # cluster_operator_up reports the health status of the core cluster
+    # (team-OLM) cluster_operator_up reports the health status of the core cluster
```

Comment:
Can you change it to `@openshift/openshift-team-olm` if you think everyone is in that team who needs to be pinged about any changes.

Reply:
Does this have to be github handle or can we add slack user group handles? Example: @olmcop